package.json

```diff
@@ -57,7 +57,8 @@
     "express": "^5.1.0",
     "jszip": "^3.10.1",
     "node-fetch": "^3.3.2",
-    "ws": "^8.18.3"
+    "ws": "^8.18.3",
+    "express-rate-limit": "^7.1.5"
```

Comment:
Adding `express-rate-limit` should probably be done in a separate PR but when this is closer to merging, I can look at it. Probably not a bad idea to implement it.